<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HUgi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wIULLI</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="drawMazeButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Maze</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
